### DISCUSSION TITLE
abnormal peaks in flow, 2200m3/h?

### DISCUSSION THREAD

**Discussion by House8000** [wmbusmeters/wmbusmeters#977] [General]
Hi, I have a Diehl Hydrus 171A with working decryption key but I often get abnormal peaks in flow.
See attached picture.
![2023-06-11 wmbusmeters 2200 01E](https://github.com/wmbusmeters/wmbusmeters/assets/112964814/5303cb30-33ad-4227-9798-dcaccb671d6d)

What could be the problem?

`pi@RaspiOMS01:~ $ cat /etc/wmbusmeters.conf 
loglevel=normal
# Remember to change auto here to the device you are going to use in production.
device=/dev/ttyUSB0:cul:t1
logtelegrams=false
format=json
meterfiles=/var/lib/wmbusmeters/meter_readings
meterfilesaction=overwrite
logfile=/var/log/wmbusmeters/wmbusmeters.log
#shell=/usr/bin/mosquitto_pub -h 192.168.5.78 -p 1883 -u USER -P XXX -t wmbusmeters/$METER_NAME/JSON -m $METER_JSON
shell=/usr/bin/mosquitto_pub -h 192.168.5.78 -p 1883 -u USER -P XXX -t wmbusmeters/$METER_NAME/Gesamt -m $METER_TOTAL_M3
shell=/usr/bin/mosquitto_pub -h 192.168.5.78 -p 1883 -u USER -P XXX -t wmbusmeters/$METER_NAME/Durchfluss -m $METER_MAX_FLOW_M3H
shell=/usr/bin/mosquitto_pub -h 192.168.5.78 -p 1883 -u USER -P XXX -t wmbusmeters/$METER_NAME/Temperatur -m $METER_FLOW_TEMPERATURE_C
shell=/usr/bin/mosquitto_pub -h 192.168.5.78 -p 1883 -u USER -P XXX -t wmbusmeters/$METER_NAME/Status -m "$METER_STATUS"
shell=/usr/bin/mosquitto_pub -h 192.168.5.78 -p 1883 -u USER -P XXX -t wmbusmeters/$METER_NAME/Timestamp -m $METER_TIMESTAMP
shell=/usr/bin/mosquitto_pub -h 192.168.5.78 -p 1883 -u USER -P XXX -t wmbusmeters/$METER_NAME/BatterieLaufzeit -m $METER_REMAINING_BATTERY_LIFE_Year
pi@RaspiOMS01:~ $ `

`pi@RaspiOMS01:~ $ cat /etc/wmbusmeters.d/GWT-Wasser-Haus 
name=GWT-Wasser-Haus
type=hydrus:t1
id=75247347
key=XXX
pi@RaspiOMS01:~ $ 
`

Did some reboots today, also upgraded to latest version today (git pull)
`pi@RaspiOMS01:~ $ cat /var/log/wmbusmeters/wmbusmeters.log
(wmbusmeters) shutting down
(wmbusmeters) HUP received, restarting and reloading config files.
(wmbusmeters) logging started 2023-06-11 00:00:02 using 1.8.0-17-g1653899
[2023-06-11_00:00:02] Started config cul on /dev/ttyUSB0 listening on t1
(wmbusmeters) waiting for telegrams
(wmbusmeters) started meter 1 (GWT-Wasser-Haus 75247347 hydrus)
[2023-06-11_06:16:15] (memory) rss 3493888 peak 3.33 MiB
(wmbusmeters) shutting down
(wmbusmeters) logging started 2023-06-11 07:46:43 using 1.8.0-17-g1653899
[2023-06-11_07:46:44] Started config cul on /dev/ttyUSB0 listening on t1
(wmbusmeters) waiting for telegrams
[2023-06-11_07:46:46] (memory) rss 1966080 peak 1.87 MiB
(wmbusmeters) started meter 1 (GWT-Wasser-Haus 75247347 hydrus)
(wmbusmeters) shutting down
(wmbusmeters) logging started 2023-06-11 07:54:10 using 1.9.0-119-g69d4445
[2023-06-11_07:54:11] Started config cul on /dev/ttyUSB0 listening on t1
(wmbusmeters) waiting for telegrams
[2023-06-11_07:54:13] (memory) rss 1957888 peak 1.86 MiB
(wmbusmeters) started meter 1 (GWT-Wasser-Haus 75247347 auto)
(wmbusmeters) shutting down
(wmbusmeters) logging started 2023-06-11 08:23:16 using 1.9.0-119-g69d4445
[2023-06-11_08:23:17] Started config cul on /dev/ttyUSB0 listening on t1
(wmbusmeters) waiting for telegrams
[2023-06-11_08:23:19] (memory) rss 1908736 peak 1.82 MiB
(wmbusmeters) started meter 1 (GWT-Wasser-Haus 75247347 auto)
pi@RaspiOMS01:~ $ 
`

Thanks,
Thomas

**Comment by weetmuts** [comment]
Can you collect some telegrams with --logtelegrams? Can it be backward flow that is misintepreted?

**Comment by House8000** [comment]
Hi, thanks, yes i think it could be a little backward flow. Do you know if this a known behaviour for little backflow?

I edited  wmbusmters.conf with logtelegrams=true, is this Ok?
Do you need to know when the abnormal reading happened to correlate with the telegrams?


`(wmbusmeters) logging started 2023-06-11 10:37:44 using 1.9.0-119-g69d4445
[2023-06-11_10:37:45] Started config cul on /dev/ttyUSB0 listening on t1
(wmbusmeters) waiting for telegrams
[2023-06-11_10:37:48] (memory) rss 1970176 peak 1.87 MiB
(wmbusmeters) started meter 1 (GWT-Wasser-Haus 75247347 auto)
telegram=|6344A5114773247570078C0035900F002C251E781B0029EB09C66E2BFDD27A1E004007102F2F_01FD08AD0C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025ABF00C4016D3B17FF25CC011340120100|+5
telegram=|6344A5114773247570078C0035900F002C251F781B0032AF1B68E0B979127A1F004007102F2F_01FD08AE0C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025ABF00C4016D3B17FF25CC011340120100|+19
telegram=|6344A5114773247570078C0035900F002C2520781B00D96AFA6D0215C7997A20004007102F2F_01FD08AF0C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025ABF00C4016D3B17FF25CC011340120100|+34
telegram=|6344A5114773247570078C0035900F002C2521781B005F124653ACD6702C7A21004007102F2F_01FD08B00C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025ABF00C4016D3B17FF25CC011340120100|+49
telegram=|6344A5114773247570078C0035900F002C2522781B00EA497358F9154E427A22004007102F2F_01FD08B10C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025ABF00C4016D3B17FF25CC011340120100|+63
telegram=|6344A5114773247570078C0035900F002C2525781B0007F0BFC9FFB6D7E57A25004007102F2F_01FD08B40C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025ABF00C4016D3B17FF25CC011340120100|+107
telegram=|6344A5114773247570078C0035900F002C2526781B003562B5EA5F7E195B7A26004007102F2F_01FD08B50C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025AC000C4016D3B17FF25CC011340120100|+122
telegram=|6344A5114773247570078C0035900F002C2527781B004C23B56B3E52B4BC7A27004007102F2F_01FD08B60C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025AC000C4016D3B17FF25CC011340120100|+137
telegram=|6344A5114773247570078C0035900F002C2528781B007A11C55FA12053147A28004007102F2F_01FD08B70C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025AC000C4016D3B17FF25CC011340120100|+151
telegram=|6344A5114773247570078C0035900F002C252A781B00A730883D5CACE3D07A2A004007102F2F_01FD08B90C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025AC000C4016D3B17FF25CC011340120100|+181
telegram=|6344A5114773247570078C0035900F002C252C781B0068B3746BF2F8C1927A2C004007102F2F_01FD08BB0C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025AC000C4016D3B17FF25CC011340120100|+195
telegram=|6344A5114773247570078C0035900F002C252D781B00473426184AC85F947A2D004007102F2F_01FD08BD0C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025AC000C4016D3B17FF25CC011340120100|+210
telegram=|6344A5114773247570078C0035900F002C252E781B00D70C9090514EC6867A2E004007102F2F_01FD08BF0C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025AC000C4016D3B17FF25CC011340120100|+225
telegram=|6344A5114773247570078C0035900F002C252F781B00087FC0779075BCC77A2F004007102F2F_01FD08C10C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025AC000C4016D3B17FF25CC011340120100|+239
telegram=|6344A5114773247570078C0035900F002C2530781B00B069573509443B557A30004007102F2F_01FD08C30C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025AC000C4016D3B17FF25CC011340120100|+254
telegram=|6344A5114773247570078C0035900F002C2531781B00E52C570249461DC27A31004007102F2F_01FD08C50C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025AC000C4016D3B17FF25CC011340120100|+268
telegram=|6344A5114773247570078C0035900F002C2532781B00A0B70FF359B070447A32004007102F2F_01FD08C60C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025AC000C4016D3B17FF25CC011340120100|+283
telegram=|6344A5114773247570078C0035900F002C2533781B009BE98E9C13AAC8C47A33004007102F2F_01FD08C70C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025AC000C4016D3B17FF25CC011340120100|+298
telegram=|6344A5114773247570078C0035900F002C2536781B00DB56171291DEC8A07A36004007102F2F_01FD08CA0C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025AC000C4016D3B17FF25CC011340120100|+342
telegram=|6344A5114773247570078C0035900F002C2537781B00D68EFBE8E3ADC2DB7A37004007102F2F_01FD08CB0C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025AC000C4016D3B17FF25CC011340120100|+357
telegram=|6344A5114773247570078C0035900F002C2538781B000B26C3296243392D7A38004007102F2F_01FD08CC0C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025AC000C4016D3B17FF25CC011340120100|+371
telegram=|6344A5114773247570078C0035900F002C2539781B0017FC1A5136BAA47A7A39004007102F2F_01FD08CD0C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025AC000C4016D3B17FF25CC011340120100|+386
telegram=|6344A5114773247570078C0035900F002C253B781B00C25B1420E0DAAD897A3B004007102F2F_01FD08CF0C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025AC000C4016D3B17FF25CC011340120100|+415
telegram=|6344A5114773247570078C0035900F002C253C781B006FD5F7A35494D97F7A3C004007102F2F_01FD08D00C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025AC000C4016D3B17FF25CC011340120100|+430
telegram=|6344A5114773247570078C0035900F002C253E781B00B9FB86C23540FDAB7A3E004007102F2F_01FD08D20C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025AC000C4016D3B17FF25CC011340120100|+459
telegram=|6344A5114773247570078C0035900F002C2540781B0008131EE7E6FC29C27A40004007102F2F_01FD08D40C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025AC000C4016D3B17FF25CC011340120100|+489
telegram=|6344A5114773247570078C0035900F002C2542781B009C72AAEC0E49398C7A42004007102F2F_01FD08D60C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025AC100C4016D3B17FF25CC011340120100|+503
telegram=|6344A5114773247570078C0035900F002C2543781B004C84D2108C4F82C77A43004007102F2F_01FD08D80C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025AC100C4016D3B17FF25CC011340120100|+518
telegram=|6344A5114773247570078C0035900F002C2544781B0009386B173771C85F7A44004007102F2F_01FD08DA0C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025AC100C4016D3B17FF25CC011340120100|+532
telegram=|6344A5114773247570078C2036900F002C2547781B00825AA3BEF42595AA7A47004007102F2F_01FD08DF0C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025AC100C4016D3B17FF25CC011340120100|+566
telegram=|6344A5114773247570078C0036900F002C254A781B003276A6CBA3EA6AC47A4A004007102F2F_01FD08E20C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025AC100C4016D3B17FF25CC011340120100|+606
telegram=|6344A5114773247570078C0036900F002C254B781B00AEA4AB11B88E8A197A4B004007102F2F_01FD08E30C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025AC100C4016D3B17FF25CC011340120100|+620
telegram=|6344A5114773247570078C0036900F002C254D781B00AE52385ACF671D657A4D004007102F2F_01FD08E50C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025AC100C4016D3B17FF25CC011340120100|+650
telegram=|6344A5114773247570078C0036900F002C254E781B00CA0325C97F45A4357A4E004007102F2F_01FD08E60C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025AC100C4016D3B17FF25CC011340120100|+665
telegram=|6344A5114773247570078C0036900F002C254F781B0046B5B14155D0CBAB7A4F004007102F2F_01FD08E70C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025AC100C4016D3B17FF25CC011340120100|+679
telegram=|6344A5114773247570078C0036900F002C2551781B001BD788C35CBE81F97A51004007102F2F_01FD08E90C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025AC100C4016D3B17FF25CC011340120100|+709
telegram=|6344A5114773247570078C0036900F002C2555781B00F729EF83D5B1B58A7A55004007102F2F_01FD08ED0C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025AC100C4016D3B17FF25CC011340120100|+768
telegram=|6344A5114773247570078C0036900F002C2556781B0019BBFAE1B42CF13E7A56004007102F2F_01FD08EE0C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025AC100C4016D3B17FF25CC011340120100|+782
telegram=|6344A5114773247570078C0036900F002C2558781B0028FCE9398C166D247A58004007102F2F_01FD08F00C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025AC100C4016D3B17FF25CC011340120100|+797
telegram=|6344A5114773247570078C0036900F002C255A781B007B6B9E4536DABF417A5A004007102F2F_01FD08F40C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025AC200C4016D3B17FF25CC011340120100|+826
telegram=|6344A5114773247570078C0036900F002C255C781B00DF59B2C1D26B70607A5C004007102F2F_01FD08F80C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025AC200C4016D3B17FF25CC011340120100|+855
telegram=|6344A5114773247570078C0036900F002C255E781B0014A63641FFC8FEF47A5E004007102F2F_01FD08FB0C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025AC200C4016D3B17FF25CC011340120100|+885
telegram=|6344A5114773247570078C0036900F002C2560781B00AB965F41252EB9E77A60004007102F2F_01FD08FD0C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025AC200C4016D3B17FF25CC011340120100|+914
telegram=|6344A5114773247570078C0036900F002C2562781B00C9AF2CA1FFD7BDA87A62004007102F2F_01FD08FF0C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025AC200C4016D3B17FF25CC011340120100|+943
telegram=|6344A5114773247570078C0036900F002C2563781B0096CC64FE9E4533FF7A63004007102F2F_01FD08000C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025AC200C4016D3B17FF25CC011340120100|+958
telegram=|6344A5114773247570078C0036900F002C2566781B00CE13CCFC21B718DD7A66004007102F2F_01FD08030C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025AC200C4016D3B17FF25CC011340120100|+1002
telegram=|6344A5114773247570078C0036900F002C2567781B001E3FC0939C8F3FC27A67004007102F2F_01FD08040C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025AC200C4016D3B17FF25CC011340120100|+1017
telegram=|6344A5114773247570078C0036900F002C2568781B00E9648149FD8BCBDE7A68004007102F2F_01FD08050C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025AC200C4016D3B17FF25CC011340120100|+1031
telegram=|6344A5114773247570078C0036900F002C2569781B004F62F7CF626FEE407A69004007102F2F_01FD08060C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025AC200C4016D3B17FF25CC011340120100|+1046
telegram=|6344A5114773247570078C0036900F002C256B781B00BD71787FD88F0CD87A6B004007102F2F_01FD08080C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025AC200C4016D3B17FF25CC011340120100|+1075
telegram=|6344A5114773247570078C0036900F002C256C781B00D44CE5DB91EC66E57A6C004007102F2F_01FD08090C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025AC200C4016D3B17FF25CC011340120100|+1090
telegram=|6344A5114773247570078C0036900F002C256E781B00799A59999ED82DE27A6E004007102F2F_01FD080B0C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025AC200C4016D3B17FF25CC011340120100|+1104
telegram=|6344A5114773247570078C0036900F002C2572781B0023AB1209FC3C34087A72004007102F2F_01FD08130C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025AC300C4016D3B17FF25CC011340120100|+1163
telegram=|6344A5114773247570078C0036900F002C2573781B00BA416514A33C8F5D7A73004007102F2F_01FD08150C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025AC300C4016D3B17FF25CC011340120100|+1178
telegram=|6344A5114773247570078C0036900F002C2574781B0005FD22A2D9AFE1F67A74004007102F2F_01FD08160C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025AC300C4016D3B17FF25CC011340120100|+1193
telegram=|6344A5114773247570078C0036900F002C2575781B0001A1A99FD71678207A75004007102F2F_01FD08170C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025AC300C4016D3B17FF25CC011340120100|+1207
telegram=|6344A5114773247570078C0036900F002C2576781B008917A81521A497607A76004007102F2F_01FD08180C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025AC300C4016D3B17FF25CC011340120100|+1222
telegram=|6344A5114773247570078C0036900F002C2577781B00DAD4ACC21671AFAF7A77004007102F2F_01FD08190C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025AC300C4016D3B17FF25CC011340120100|+1237
telegram=|6344A5114773247570078C0036900F002C2578781B008FDF30F206D1BB6D7A78004007102F2F_01FD081A0C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025AC300C4016D3B17FF25CC011340120100|+1252
telegram=|6344A5114773247570078C0036900F002C2579781B003278996A4408DA677A79004007102F2F_01FD081B0C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025AC300C4016D3B17FF25CC011340120100|+1266
telegram=|6344A5114773247570078C0036900F002C257A781B005BEBBEB694A74DD37A7A004007102F2F_01FD081C0C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025AC300C4016D3B17FF25CC011340120100|+1281
telegram=|6344A5114773247570078C0036900F002C257C781B0085AE209BFC7600B87A7C004007102F2F_01FD081E0C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025AC300C4016D3B17FF25CC011340120100|+1310
telegram=|6344A5114773247570078C0036900F002C257D781B0001C2B8DB029D5A207A7D004007102F2F_01FD081F0C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025AC300C4016D3B17FF25CC011340120100|+1325
telegram=|6344A5114773247570078C0036900F002C2580781B0007465300405BE9A87A80004007102F2F_01FD08220C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025AC300C4016D3B17FF25CC011340120100|+1369
telegram=|6344A5114773247570078C0036900F002C2584781B000BE7695C2F50D2AC7A84004007102F2F_01FD08270C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025AC300C4016D3B17FF25CC011340120100|+1413
telegram=|6344A5114773247570078C0036900F002C2585781B00E7C4CF56AD22B1227A85004007102F2F_01FD08290C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025AC300C4016D3B17FF25CC011340120100|+1427
telegram=|6344A5114773247570078C0036900F002C2586781B00F89CBE7C8F2F85447A86004007102F2F_01FD082B0C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025AC300C4016D3B17FF25CC011340120100|+1442
telegram=|6344A5114773247570078C2037900F002C2588781B001F2BC890DD32BD027A88004007102F2F_01FD082E0C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025AC300C4016D3B17FF25CC011340120100|+1470
telegram=|6344A5114773247570078C0037900F002C2589781B009BD2A7917B5CC0737A89004007102F2F_01FD082F0C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025AC300C4016D3B17FF25CC011340120100|+1472
telegram=|6344A5114773247570078C0037900F002C258A781B009DBC7B024E7C2B187A8A004007102F2F_01FD08300C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025AC300C4016D3B17FF25CC011340120100|+1486
telegram=|6344A5114773247570078C0037900F002C258C781B00370EA2436033DAFD7A8C004007102F2F_01FD08320C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025AC300C4016D3B17FF25CC011340120100|+1516
telegram=|6344A5114773247570078C0037900F002C258D781B00AEE3B5E5D68BCC3C7A8D004007102F2F_01FD08330C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025AC300C4016D3B17FF25CC011340120100|+1530
telegram=|6344A5114773247570078C0037900F002C258E781B00DBF84883CD26AB967A8E004007102F2F_01FD08340C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025AC300C4016D3B17FF25CC011340120100|+1545
telegram=|6344A5114773247570078C0037900F002C258F781B0005AF0AE5C52C421C7A8F004007102F2F_01FD08350C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025AC300C4016D3B17FF25CC011340120100|+1559
telegram=|6344A5114773247570078C0037900F002C2590781B00D6FE7D2F1B72EE1D7A90004007102F2F_01FD08360C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025AC300C4016D3B17FF25CC011340120100|+1574
telegram=|6344A5114773247570078C0037900F002C2591781B00162F6BDDB312DA017A91004007102F2F_01FD08370C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025AC400C4016D3B17FF25CC011340120100|+1589
telegram=|6344A5114773247570078C0037900F002C2592781B00B5E0B0816291A1687A92004007102F2F_01FD08380C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025AC400C4016D3B17FF25CC011340120100|+1603
telegram=|6344A5114773247570078C0037900F002C2593781B00B9413E75498CABEA7A93004007102F2F_01FD08390C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025AC400C4016D3B17FF25CC011340120100|+1618
telegram=|6344A5114773247570078C0037900F002C2594781B0009A4833BFEF635907A94004007102F2F_01FD083A0C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025AC400C4016D3B17FF25CC011340120100|+1633
telegram=|6344A5114773247570078C0037900F002C2595781B001BFDB2493F8BC6F57A95004007102F2F_01FD083B0C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025AC400C4016D3B17FF25CC011340120100|+1648
telegram=|6344A5114773247570078C0037900F002C2596781B00F6B6B99BE88FDA387A96004007102F2F_01FD083C0C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025AC400C4016D3B17FF25CC011340120100|+1663
telegram=|6344A5114773247570078C0037900F002C2597781B003B7EDC6C9F17CC357A97004007102F2F_01FD083D0C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025AC400C4016D3B17FF25CC011340120100|+1677
telegram=|6344A5114773247570078C0037900F002C2598781B00D58A141ED0CC2D5F7A98004007102F2F_01FD083E0C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025AC400C4016D3B17FF25CC011340120100|+1692
telegram=|6344A5114773247570078C0037900F002C259A781B009747DE726E0856217A9A004007102F2F_01FD08400C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025AC400C4016D3B17FF25CC011340120100|+1706
telegram=|6344A5114773247570078C0037900F002C259B781B007934A6644A4595BA7A9B004007102F2F_01FD08420C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025AC400C4016D3B17FF25CC011340120100|+1721
telegram=|6344A5114773247570078C0037900F002C259C781B00F4118C730716120C7A9C004007102F2F_01FD08440C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025AC400C4016D3B17FF25CC011340120100|+1735
telegram=|6344A5114773247570078C0037900F002C259D781B00990E93BB18706C947A9D004007102F2F_01FD08460C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025AC400C4016D3B17FF25CC011340120100|+1750
telegram=|6344A5114773247570078C0037900F002C259E781B006649CCBE1D6F43627A9E004007102F2F_01FD08480C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025AC400C4016D3B17FF25CC011340120100|+1765
telegram=|6344A5114773247570078C0037900F002C25A0781B00DB22297C360F851C7AA0004007102F2F_01FD084B0C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025AC400C4016D3B17FF25CC011340120100|+1794
telegram=|6344A5114773247570078C0037900F002C25A1781B008BF244C188F0EE7A7AA1004007102F2F_01FD084C0C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025AC400C4016D3B17FF25CC011340120100|+1809
telegram=|6344A5114773247570078C0037900F002C25A2781B0098B17DFCE52D9A647AA2004007102F2F_01FD084D0C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025AC400C4016D3B17FF25CC011340120100|+1824
telegram=|6344A5114773247570078C0037900F002C25A3781B007F40F607A9786CC67AA3004007102F2F_01FD084E0C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025AC400C4016D3B17FF25CC011340120100|+1839
telegram=|6344A5114773247570078C0037900F002C25A4781B0005B7F23C2C61473D7AA4004007102F2F_01FD084F0C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025AC400C4016D3B17FF25CC011340120100|+1853
telegram=|6344A5114773247570078C0037900F002C25A5781B008D35A7C0E19C89377AA5004007102F2F_01FD08500C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025AC400C4016D3B17FF25CC011340120100|+1868
telegram=|6344A5114773247570078C0037900F002C25A7781B0001EA5DBC82EE18857AA7004007102F2F_01FD08520C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025AC400C4016D3B17FF25CC011340120100|+1897
telegram=|6344A5114773247570078C0037900F002C25A8781B00DD62E589F3A7E87B7AA8004007102F2F_01FD08530C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025AC400C4016D3B17FF25CC011340120100|+1912
telegram=|6344A5114773247570078C0037900F002C25AA781B000A6093D60939D28B7AAA004007102F2F_01FD08550C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025AC500C4016D3B17FF25CC011340120100|+1941
telegram=|6344A5114773247570078C0037900F002C25AC781B008D609E45DB90F8C67AAC004007102F2F_01FD08570C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025AC500C4016D3B17FF25CC011340120100|+1971
telegram=|6344A5114773247570078C0037900F002C25AF781B00BD4FF358A9FA0BFB7AAF004007102F2F_01FD085A0C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025AC500C4016D3B17FF25CC011340120100|+2000
telegram=|6344A5114773247570078C0037900F002C25B0781B003748958A1FB380747AB0004007102F2F_01FD085C0C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025AC500C4016D3B17FF25CC011340120100|+2015
telegram=|6344A5114773247570078C0037900F002C25B1781B00089919C9C662AF887AB1004007102F2F_01FD085E0C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025AC500C4016D3B17FF25CC011340120100|+2029
telegram=|6344A5114773247570078C0037900F002C25B2781B005BABCA8126AE29FC7AB2004007102F2F_01FD08600C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025AC500C4016D3B17FF25CC011340120100|+2043
telegram=|6344A5114773247570078C0037900F002C25B3781B0056D97E2FC505D3157AB3004007102F2F_01FD08620C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025AC500C4016D3B17FF25CC011340120100|+2058
telegram=|6344A5114773247570078C0037900F002C25B4781B00D12B65CD8C6CF5B57AB4004007102F2F_01FD08640C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025AC500C4016D3B17FF25CC011340120100|+2073
telegram=|6344A5114773247570078C0037900F002C25B5781B00B5B69366200644757AB5004007102F2F_01FD08650C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025AC500C4016D3B17FF25CC011340120100|+2088
telegram=|6344A5114773247570078C0037900F002C25B6781B00888E08F406379D8F7AB6004007102F2F_01FD08660C13411901004C1309110100CC101300000000CC201309110100426CDF2C0B3B00000002FD742412025AC500C4016D3B17FF25CC011340120100|+2102
`
